server/mysterium_api.go

```diff
@@ -98,9 +99,12 @@ func (mApi *mysteriumAPI) UnregisterProposal(proposal dto_discovery.ServicePropo
 
 // PingProposal pings service proposal as being alive
 func (mApi *mysteriumAPI) PingProposal(proposal dto_discovery.ServiceProposal, signer identity.Signer) error {
-	req, err := requests.NewSignedPostRequest(mApi.discoveryAPIAddress, "ping_proposal", dto.NodeStatsRequest{
+	statsDto := dto.NodeStatsRequest{
 		NodeKey: proposal.ProviderID,
-	}, signer)
+		NodeVersion: version.GetInfo().BuildNumber,
```

Comment:
BuildNumber is not NodeVersion. Node version is more semantic version which comes to GetInfo().Branch field (i.e. tag was 0.1.16)

Reply:
rewrote NodeVersion in such format: "branch(build number)"